src/components/HTMLEngineProvider/BaseHTMLEngineProvider.tsx

```diff
@@ -79,7 +80,17 @@ function BaseHTMLEngineProvider({textSelectable = false, children, enableExperim
             }),
             strong: HTMLElementModel.fromCustomModel({
                 tagName: 'strong',
-                mixedUAStyles: {whiteSpace: 'pre'},
+                getMixedUAStyles: (tnode) => (isChildOfTaskTitle(tnode as TNode) ? {} : styles.strong),
+                contentModel: HTMLContentModel.textual,
+            }),
+            em: HTMLElementModel.fromCustomModel({
+                tagName: 'em',
+                getMixedUAStyles: (tnode) => (isChildOfTaskTitle(tnode as TNode) ? styles.taskTitleMenuItemItalic : styles.em),
+                contentModel: HTMLContentModel.textual,
+            }),
+            h1: HTMLElementModel.fromCustomModel({
+                tagName: 'h1',
+                getMixedUAStyles: (tnode) => (isChildOfTaskTitle(tnode as TNode) ? {} : styles.h1),
```

Comment:
I leave the empty style here if `isChildOfTaskTitle` is true for `strong` and `h1` because the previous implementation added a `taskTitleMenuItem` style which is already available from the existing `task-title` style above.